### ISSUE TITLE
Ledger Nano X - Unable to add collateral

### ISSUE THREAD

**Issue by hammertime71** [Berry-Pool/nami-wallet#529]
I cannot seem to add collateral to interact with smart contracts.  The UI says connect ledger to do this but the button to Confirm remains greyed out despite the ledger being connected and the ADA app being open.  Other than this, the legder integration is awesome!

**Comment by Faghe** [comment]
I have the same problem with the same wallet!

**Comment by Faghe** [comment]
Did you manage to solve the issue?

**Comment by hammertime71** [comment]
Nope, U?

**Comment by Faghe** [comment]
Not at all. I wanted to try to reinstall and unistall. Maybe adding the
collateral when the Nami wallet is empty will do.

I'm not sure but without that funciont it's not really useful.
Regards,

**Comment by hammertime71** [comment]
What I do at the moment is have a transactional wallet and the ledger.  The ledger is my Hodl wallet and when I need to do anything I send funds to the other wallet in Nami.  This is super easy and super quick.  You literally just choose the other wallet from the dropdown.  It is a decent workaround in the meantime.  The normal wallet interacts with the smart contracts but you still have your main holdings safe and secure with the ledger private key.

**Comment by Faghe** [comment]
Hi,

I agree it's ok but then if you need to stake a considerable amount, for
example I have meld tokens from the ispo, then you need to rely on the nami
wallet seed. Which is fine but still ledger is totally bypassed.
Regards,

**Comment by hammertime71** [comment]
Yes that sucks.  My smart contract interactions are purely for buying NFT's at the moment so they are in and out.  I hope someone looks at this thread and sorts it out.

**Comment by katakombi** [comment]
Now that ledger released version 4.0.0 of their Cardano app it should be technically possible to support smart contracts and collateral. Two things have to happen: users have to upgrade their Cardano app using ledger live / NAMI has to implement the feature
